packages/riverpod_lint_flutter_test/test/goldens/lints/missing_build_method.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+// expect_lint: missed_build_method
```

Comment:
Could you add:
- A case where a class has no build method and is not annotated by riverpod
- a class with no build method but one non-build method
- a class that correctly does have a build method